querybook/server/lib/notify/notifier/slack_notifier.py

```diff
@@ -18,7 +18,8 @@ def notifier_format(self):
         return "plaintext"
 
     def notify(self, user, message):
-        to = f"@{user.username}"
+        is_old_format = user.username.startswith("@") or user.username.startswith("#")
+        to = f"{user.username}" if is_old_format else f"@{user.username}"
```

Comment:
@jczhong84 @czgu I have done that a user can set channel or user using '#' or '@' accordingly. But, we should think about the backward compatibility and I made it like before, just convert data. But maybe we should just add a migration that can transform this value in the database?